easybuild/easyconfigs/c/CPMD/CPMD-4.1r3816-intel-2015a.eb

```diff
@@ -0,0 +1,23 @@
+name = 'CPMD'
+version = '4.1'
+
+homepage = 'http://cpmd.org'
+description = """The CPMD code is a parallelized plane wave / pseudopotential
+implementation of Density Functional Theory, particularly designed for
+ab-initio molecular dynamics."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://cpmd.org/download']
+patches = ['%(name)s-%(version)sr3812.patch', '%(name)s-%(version)sr3813.patch', '%(name)s-%(version)sr3816.patch']
+prefix_opt = '-DEST='
+
+sanity_check_paths = {
+      'files': ['bin/cpmd.x', 'lib/libcpmd.a'],
+      'dirs': ['bin', 'lib'],
+}
+
+group = 'cpmd'
```

Comment:
well, auto-download won't work, so that's good enough?

you can consider adding an message-on-load:

```python
modloadmsg = "Note: authorised users only (%(name)s)"
```